db-adapter/model/src/main/java/by/gendzucuuu/crypto/currency/watcher/db/adapter/model/request/CryptocoinRequestDto.java

```diff
@@ -0,0 +1,21 @@
+package by.gendzucuuu.crypto.currency.watcher.db.adapter.model.request;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CryptocoinRequestDto {
+    private Long id;
+    private String symbol;
```

Comment:
@NotNull and @Valid?=) if you really wont, you can check does symbol correct or not))